Docs/test agent reference

iryabov opened the pull request.

iryabov committed:
docs: add test-agent reference

EPMDJ-10920

iryabov committed:
docs: add some agent parameters

EPMDJ-10921

RomanDavlyatshin commented on versioned_docs/version-0.9.0/agents/test-agent.mdx:
```diff
@@ -5,6 +5,163 @@ title: Test Agent
+General integration steps are:
+
+1. [Download Drill4J agent files](#how-to-download)
+2. [Set path to agent when running tests](#how-to-set-path-to-agent)
```
link does not work

RomanDavlyatshin commented on versioned_docs/version-0.9.0/agents/test-agent.mdx:
```diff
@@ -5,6 +5,163 @@ title: Test Agent
+    - `drill-runtime.jar`
+    - `drill.properties`
+
+Refer to [next section](/docs/agents/autotest-agent#how-to-set-path-to-agent) for further steps.
```
link does not work

RomanDavlyatshin commented on versioned_docs/version-0.9.0/agents/test-agent.mdx:
```diff
@@ -5,6 +5,163 @@ title: Test Agent
+
+Refer to [next section](/docs/agents/autotest-agent#how-to-set-path-to-agent) for further steps.
+
+### Use Drill4J CI/CD Integration Plugin for Maven and Gradle
```
I suggest to switch this with `Download agent files manually` and highlight that this is the primary way to use Test Agent

iryabov committed:
docs: fix links

iryabov committed:
Merge remote-tracking branch 'origin/main' into docs/test-agent-reference

# Conflicts:
#	versioned_docs/version-0.9.0/cicd-integration/github.mdx

iryabov committed:
Merge remote-tracking branch 'origin/main' into docs/test-agent-reference

iryabov committed:
docs: add the Error "Could not create an Appender of type ..."
docs: remove known compatibility issue with Mac Arc64 as it is no longer relevant
